eng/common/AzurePipelineTemplates/Maestro-PublishBuildToMaestro-Steps.yml

```diff
@@ -0,0 +1,168 @@
+parameters:
+  AssetName: ''
+  AssetVersion: ''
+  TriggerSubscription: true
+

```

Comment:
Please copy eng/common again from the main branch of ProjectReunionInternal as there has been updates.

Reply:
Thanks, done.